tests/test_qdrant_client.py

```
@@ -58,17 +61,21 @@
     init_remote,
     initialize_fixture_collection,
 )
+from tests.conversions.fixtures import match_text
```

Ruff (0.8.2)

`tests.conversions.fixtures.match_text` imported but unused

Remove unused import: `tests.conversions.fixtures.match_text`

(F401)